src/classes/frDonation.cls

```diff
@@ -53,84 +55,126 @@ public class frDonation extends frModel implements frSyncable {
         return mappings;
     }
     
-    private Opportunity o;
+    private SObject o;
+    private SObject gt;
```

Comment:
It looks like the `o` object has been made generic and is used in both branches of NPC/NPSP, do we need this GT object in that case?

Reply:
Nope not needed! removed in new update.

Reply:
It looks like this field wasn't removed, can you double check if the most recent commit was sent properly?